sos/policies/init_systems/systemd.py

```diff
@@ -43,8 +43,11 @@ def load_all_services(self):
             except IndexError:
                 pass
 
-    def is_running(self, name):
-        svc = self.get_service_status(name)
-        return svc['status'] == 'active'
+    def is_running(self, name, default=False):
+        try:
+            svc = self.get_service_status(name)
+            return svc['status'] == 'active'
+        except Exception:
+            return default
```

Comment:
Is there some known call flow that could lead to an exception we are catching here, or is it there "just in case" / to fallback to `default`?

Reply:
More of a just in case - haven't seen anything yet, but there is the potential situation in which we are able to load the systemd `InitSystem`, but actual calls to querying service statuses fails.